backend/routes/dates.js

```diff
@@ -0,0 +1,61 @@
+const router = require('express').Router();
+const { resetWatchers } = require('nodemon/lib/monitor/watch');
+const Dates = require('../models/date');
+const Users = require('../models/user');
+
+router.route('/').get((req, res) => {
+    Dates.find()     //list of all dates in Date model
+        .then(dates => res.json(dates))
+        .catch(err => res.status(400).json('Error: ' + err));
+});
+
+const userpromisefunc = (_id) => {
```

Comment:
The JS/TS naming convention for functions is camelCase

https://en.wikipedia.org/wiki/Camel_case
```suggestion
const userPromiseFunc = (_id) => {
```